packages/worker/src/workflow/bundler.ts

```diff
@@ -95,16 +94,15 @@ export class WorkflowCodeBundler {
     // Cast because the type definitions are inaccurate
     const memoryFs = memfs.createFsFromVolume(vol);
     ufs.use(memoryFs as any).use({ ...realFS, readdir: readdir as any });
-    const distDir = '/dist';
+    const distDir = '/dist'; // FIXME: Make it possible for user to specify destination path
```

Comment:
This would require writing to the real FS as well

Reply:
I removed the comment. The use case for this isn't clear enough. It would complexify the API and yet, there would be no guarantee that the file would actually be loaded from the same path location as it was created initially. For the same reason, I removed all references to `codePath` in other files.